### PR TITLE
adds a data indicator component and story

### PR DESCRIPTION
**Pull request by skuhlmann**
## GitHub Issue

[this issue](https://github.com/HausDAO/daohaus-monorepo/issues/388)

## Changes

- new molecule component
- storybook elements
- states
  - small
  - large
  - w/ Tooltip

Screenshot from 2022-07-25 14-52-39
Screenshot from 2022-07-25 14-52-33
Screenshot from 2022-07-25 14-52-25

## Checks

Before making your PR, please check the following:

- [x] Critical lint errors are resolved
- [x] App runs locally
- [x] App builds locally (run the build command for *any impacted package* and check for any errors before the PR)

### CODE REVIEW

File: libs/ui/src/components/molecules/DataIndicator/DataIndicator.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+export const DataIndicator = ({
+  label,
+  data,
+  size = 'lg',
+  info,
+}: DataIndicatorProps) => {
+  if (size === 'sm') {
+    return (
+      <DataIndicatorContainer>
+        <ParMd style={{ marginBottom: '0.5rem' }}>
```
Comment (jordanlesich) [inline libs/ui/src/components/molecules/DataIndicator/DataIndicator.tsx:20]: Let's try to stay away from inline styles if possible. Makes it harder for other devs to add their own styles later on. 

You can create create a class and target that class inside or use IndicatorMd.

File: libs/ui/src/components/molecules/DataIndicator/DataIndicator.styles.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+import { ParMd } from '../../atoms';
+
+export const DataIndicatorContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+export const IndicatorLabelMd = styled(ParMd)`
```
Comment (jordanlesich) [inline libs/ui/src/components/molecules/DataIndicator/DataIndicator.styles.tsx:9]: Is this being used anywhere?

**Review by jordanlesich — APPROVED** [review]
Looking slick! Just a couple small things to fix.

File: libs/ui/src/components/molecules/DataIndicator/DataIndicator.styles.tsx (hunk 2 above)
Comment (skuhlmann) [reply libs/ui/src/components/molecules/DataIndicator/DataIndicator.styles.tsx:9]: Oht hat was supposed to replace my inline style that you commented on above.

**Review by jonathanprozzi — APPROVED** [review]
Nice job!

**Pull request merged**